app/src/main/java/ltd/evilcorp/atox/ToxThread.kt

```diff
@@ -47,12 +48,20 @@ class ToxThread(saveDestination: String, saveOption: SaveDataOptions) : HandlerT
                     tox.setName(it.obj as String)
                     true
                 }
+
```

Comment:
Remove unrelated formatting changes. If you want to add more whitespace, I'd be fine with a separate commit doing it for the entire block or just a separate PR.

Reply:
done